redi-connect-front/src/components/organisms/Checklist.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Section, Container, Element, Columns } from 'react-bulma-components'
+import Heading from '../atoms/Heading'
+import { ReactComponent as Check } from '../../assets/images/checkmark.svg'
+
+const checks = {
+  mentor: [
+    'I work in the digital industry',
+    'I am willing to share knowledge and experiences with others',
+    'I am open to meet other nationalities',
+    'I have the time to meet mentees during my free time',
+    'I have time to support on a personal and professional level'
+  ],
+  mentee: [
+    'I am a student or student alumni at ReDI school',
+    'I would like help on my classes and/or career development',
+    'I would like to get guidance from an expert from the digital industry',
+    'I can make time to meet with a mentor on a regular basis',
+    'I am open to get feedback on the topics I disscuss with my mentor'
+  ]
+}
+
+interface CustomProps {
+  title: string
+  headline: string
+  checks?: Array<string>
+}
+
+interface DefaultProps {
+  type: 'mentor' | 'mentee'
+}
+
+const Checklist = ({ title, headline, checks }: CustomProps) => {
+  return (
+    <Section className="default-background">
+      <Container className="double-block-space">
```

Comment:
Why do you need to have two container? Just wrap the inside of the section with it

Reply:
done